### PR TITLE
Homepage + LoginPage and tests added

### PR DESCRIPTION
**Pull request by Denis-Andrei**

### CODE REVIEW

File: app/uk/gov/hmrc/examplefrontend/controllers/HomePageController.scala [hunk 1]
```diff
@@ -16,20 +16,22 @@
...
-class HelloWorldController @Inject()(
-  mcc: MessagesControllerComponents,
-  helloWorldPage: HelloWorldPage)
-    extends FrontendController(mcc) {
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
 
-  val helloWorld: Action[AnyContent] = Action.async { implicit request =>
-    Future.successful(Ok(helloWorldPage()))
+case class HomePageController @Inject()(
```
Comment (simonpforster) [inline app/uk/gov/hmrc/examplefrontend/controllers/HomePageController.scala]: Should be as
@Singleton
class ...

File: app/uk/gov/hmrc/examplefrontend/controllers/LoginController.scala [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import play.api.libs.ws.{WSClient, WSResponse}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.examplefrontend.models.{Client, User, UserForm}
+import uk.gov.hmrc.examplefrontend.views.html.LoginPage
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+case class LoginController @Inject()(
```
Comment (simonpforster) [inline app/uk/gov/hmrc/examplefrontend/controllers/LoginController.scala]: remove case

File: app/uk/gov/hmrc/examplefrontend/controllers/HomePageController.scala [hunk 3]
```diff
@@ -16,20 +16,22 @@
...
+   mcc: MessagesControllerComponents,
+   homePage: HomePage
+   )
+  extends FrontendController(mcc){
+
+  def homepage = Action.async { implicit request =>
+    Future.successful(Ok(homePage()))
```
Comment (simonpforster) [inline app/uk/gov/hmrc/examplefrontend/controllers/HomePageController.scala]: Don't need the async here, can remove both .async and Future.successful

File: app/uk/gov/hmrc/examplefrontend/controllers/LoginController.scala [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  ws: WSClient,
+  mcc: MessagesControllerComponents,
+  loginPage: LoginPage,
+  implicit val ec: ExecutionContext)
+    extends FrontendController(mcc) with I18nSupport{
+
+
+  def login: Action[AnyContent] = Action.async { implicit request =>
```
Comment (simonpforster) [inline app/uk/gov/hmrc/examplefrontend/controllers/LoginController.scala]: Same again, can remove the .async & future.successful

**Review by simonpforster — CHANGES_REQUESTED** [review]

**Review by simonpforster — APPROVED** [review]

**Review by selinahussain — APPROVED** [review]

**Pull request merged**